src/xmppjs/XJSAccount.ts

```diff
@@ -54,10 +54,17 @@ export class XmppJsAccount implements IBifrostAccount {
                             this.lastStanzaTs.set(roomName, Date.now());
                             return;
                         }
+                        const handle: string|undefined = this.roomHandles.get(roomName);
```

Comment:
Request: remove the typehint here, so that the typechecking can be eventually handled by `strictNullChecks` (if it's ever turned on at some point).